authn-oidc-files/keycloak/create_user

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Are these files in `authn-oidc-files` used in the CI process? If so, could we move them into the `ci` folder? Seems awfully specific to a certain part of Conjur to be living at the top level.

Reply:
@jtuttle they are used both in ci and in dev. that's why there in a common path. Do you still think they should be in `ci` and referred to from `dev` as `../ci/authn-oidc-files/keycloak`?

Reply:
after our conversation in slack, moving to `ci` folder